send_traces.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+mkdir -p ~/scripts
+
+# List of API endpoints
+apis=(
+"http://playground.sigscalr.io:8080/dispatch?customer=123"
+"http://playground.sigscalr.io:8080/config?customer=123"
+"http://playground.sigscalr.io:8080/dispatch?customer=392"
+"http://playground.sigscalr.io:8080/config?customer=392"
+"http://playground.sigscalr.io:8080/dispatch?customer=731"
+"http://playground.sigscalr.io:8080/config?customer=731"
+"http://playground.sigscalr.io:8080/dispatch?customer=567"
+"http://playground.sigscalr.io:8080/config?customer=567"
+)
```

Comment:
nit: add a comment saying these are the hotrod endpoints that are available on port 8080 since we install hotrod on playground with install.sh. Otherwise if we have to change this in the future it might take someone a while to figure out what these mean